src/anyvar/utils/liftover_utils.py

```diff
@@ -0,0 +1,292 @@
+
+
+def get_liftover_variant(
+    variant_object: dict, seqrepo_dataproxy: _DataProxy
+) -> VrsVariation:
+    """Liftover a variant from GRCh37 or GRCH38 into the opposite assembly, and return the converted variant as a VrsObject.
+    If liftover is unsuccessful, raise an Exception.
+
+    :param variant_object: A dictionary representation of a `VrsVariation`.
+    :param seqrepo_dataproxy: A `SeqrepoDataproxy` instance.
+    :return: The converted variant as a `VrsObject`.
+    :raises:
+        - `MalformedInputError`:  If the `variant_object` is empty or otherwise falesy
+
+        - `UnsupportedVariantLocationTypeError`: If the variant lacks a refget accession, start position or end position
+
+        - `UnsupportedReferenceAssemblyError`: If the variant's accession was not found in any supported assembly
+
+        - `AmbiguousReferenceAssemblyError`: If the variant's accession was found in multiple supported assemblies
+
+        - `ChromosomeResolutionError`: If unable to resolve variant's chromosome
+
+        - `CoordinateConversionError`: If unable to lift over the variant's start and/or end position(s)
+
+        - `AccessionConversionError`: If unable to lift over the variant's refget accession
+    """
+    if not variant_object:
+        raise MalformedInputError
+
+    # Get variant start position, end position, and refget accession - liftover is currently unsupported without these
+    refget_accession = (
+        variant_object.get("location", {})
+        .get("sequenceReference", {})
+        .get("refgetAccession")
+    )
+    start_position = variant_object.get("location", {}).get("start")
+    end_position = variant_object.get("location", {}).get("end")
+    if not refget_accession or not start_position or not end_position:
+        raise UnsupportedVariantLocationTypeError
+
+    # Determine which assembly we're converting from/to
+    prefixed_accession = f"ga4gh:{refget_accession}"
+
+    grch37 = "GRCh37"
+    grch38 = "GRCh38"
+
+    # This is required for `get_from_and_to_assemblies` and `get_chromosome_from_aliases`
+    # See function documentation for more details
+    accession_aliases = {
+        grch37: list(
+            seqrepo_dataproxy.translate_sequence_identifier(prefixed_accession, grch37)
+        ),
+        grch38: list(
+            seqrepo_dataproxy.translate_sequence_identifier(prefixed_accession, grch38)
+        ),
+    }
+
+    from_assembly = None
+    to_assembly = None
+    from_assembly, to_assembly = get_from_and_to_assemblies(
+        accession_aliases
+    )  # Will raise an `UnsupportedReferenceAssemblyError` or `AmbiguousReferenceAssemblyError` if unsuccessful
+
+    # Determine which chromosome the variant is on
+    chromosome = get_chromosome_from_aliases(accession_aliases.get(from_assembly, []))
+    if not chromosome:
+        raise ChromosomeResolutionError
+
+    # Begin liftover conversion
+    assembly_map = {grch37: Genome.HG19, grch38: Genome.HG38}
```

Comment:
similarly I think these could be bundled with an enum and/or defined as a const at module level